Dockerfile

```diff
@@ -3,39 +3,38 @@ MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
 COPY docker/centos7 /build
 
-COPY pom.xml /build/
-COPY setup.py /build
-COPY setup.cfg /build
+RUN yum -y -q install epel-release && \
+    yum -y -q install ansible
+# git is needed until openmicroscopy.omego is put on galaxy
```

Comment:
I still haven't been able to get omego to show up on galaxy. Can https://github.com/openmicroscopy/ansible-role-omego/archive/0.1.0.tar.gz be used?